### ISSUE TITLE
random media wallpaper doesn't ever change

### ISSUE THREAD

**Issue by killarny** [causefx/Organizr#1937]
###### Operating System: Ubuntu (docker)
<hr>

##### Problem Description:
I've had the "Use Random Media Wallpaper From Media Server" setting enabled for weeks now, and it doesn't ever change. It has been showing the same image the whole time of Tulsa King that it (presumably) picked up from my Plex configuration when I enabled it. As much as I like seeing Sylvester Stallone every time I log in 😁, I do have other stuff on my Plex that it could be using, but for some reason isn't.

![image](https://github.com/causefx/Organizr/assets/29001/d8081f40-8d8b-4751-9499-5423b99fde23)

<hr>

##### Reproduction Steps:
Toggle the options I have showing in that screenshot above, I guess. Sorry, I wish I could be more helpful, but I honestly have no idea what would cause this.

Is there any other information I can provide to help figure this out?

<hr>

#### Errors on screen?  If so paste here:

```

```

**Comment by causefx** [comment]
are there a lot of images inside your data/cache folder?

**Comment by killarny** [comment]
Yes, the `/config/www/organizr/data/cache` folder has 85 images in it that appear to be covers from Plex. All but one of them are `####-list.jpg` and one image is named `6044-np.jpg`.

**Comment by killarny** [comment]
Is there anything else I can provide to give insight into this issue? @causefx were you implying that I should clear the cache? (I haven't changed anything yet, in case I can be of further assistance here.)

**Comment by causefx** [comment]
it only creates -np files once someone watches a movie or item on plex while organizr is open.  i will look into fixing this. thanks.

**Comment by killarny** [comment]
Bumping so the overzealous GitHub bot doesn't close it prematurely.

**Comment by causefx** [comment]
I still haven't thought of or decided how I want to implement this yet.

**Comment by Gleaped** [comment]
It would be great if we could limit this by library, otherwise this feature is unusable.
